en/docs/cli/upgrade.md

```diff
@@ -30,6 +30,89 @@ success Saved 867 new dependencies.
 ✨  Done in 20.79s.
 ```
 
+##### `yarn upgrade [package]` <a class="toc" id="toc-yarn-upgrade-package" href="#toc-yarn-upgrade-package"></a>
+
+This upgrades a single named package to the version specified by the `latest` tag (potentially upgrading the package across major versions).
+
+```sh
+$ yarn upgrade d3-scale
+yarn upgrade vx.x.x
+[1/4] 🔍  Resolving packages...
+[2/4] 🚚  Fetching packages...
+[3/4] 🔗  Linking dependencies...
+[4/4] 📃  Building fresh packages...
+success Saved lockfile.
+success Saved 1 new dependency
+└─ d3-scale@1.0.3
+✨  Done in 6.10s.
+```
+
+This will update your `package.json` to look like this:
+
+```diff
+-  "d3-scale": "^0.9.3",
++  "d3-scale": "^1.0.3",
+```
+
+##### `yarn upgrade [package@version]` <a class="toc" id="toc-yarn-upgrade-package-version" href="#toc-yarn-upgrade-package-version"></a>
+
+This will upgrade (or downgrade) an installed package to the specified version. You can use any semver-valid version number or range.
+
+```sh
+$ yarn upgrade d3-scale@1.0.2
+yarn upgrade vx.x.x
+[1/4] 🔍  Resolving packages...
+[2/4] 🚚  Fetching packages...
+[3/4] 🔗  Linking dependencies...
+[4/4] 📃  Building fresh packages...
+success Saved lockfile.
+success Saved 1 new dependency
+└─ d3-scale@1.0.2
+✨  Done in 6.43s.
+```
+
+This will update your `package.json` to look like this:
+
+```diff
+-  "d3-scale": "^1.0.3",
++  "d3-scale": "^1.0.2",
+```
+
+Similarly, running
+
+##### `yarn upgrade [package@tag]` <a class="toc" id="toc-yarn-upgrade-package-tag" href="#toc-yarn-upgrade-package-tag"></a>
+
+This will upgrade a package to the version identified by `tag`. [Tag](./tag#toc-what-are-tags) names are chosen by project maintainers, typically you use this command to install an experimental or long term support release of an actively developed pacakge. The tag you choose will be the version that appears in your `package.json` file.
+
+```sh
+yarn upgrade react@next
+yarn upgrade v0.16.0
+[1/4] 🔍  Resolving packages...
+[2/4] 🚚  Fetching packages...
+[3/4] 🔗  Linking dependencies...
+[4/4] 📃  Building fresh packages...
+success Saved lockfile.
+success Saved 1 new dependency
+└─ react@15.4.0-rc.4
+✨  Done in 3.73s.
+```
+
+This will update your `package.json` to look like
+
+```diff
+-  "react": "^15.3.2",
++  "react": "next",
+```
+
+Similarly, using the `latest` tag will result in an updated `package.json` that looks like
+
+```diff
+-  "react": "^15.3.2",
++  "react": "latest",
+```
+
 Also see:
 
 - [`yarn add`](./add): add a package to use in your current package.
+- [`yarn tag`](./tag#toc-what-are-tags): What are tags?
+- [Versions of dependencies](../dependency-versions#toc-semantic-versioning): Semantic versioning
```

Comment:
Can you add all of these to a `additional_reading` field in the jekyll frontmatter? It makes this appear:

![screen shot 2016-10-19 at 2 28 35 pm](https://cloud.githubusercontent.com/assets/952783/19538319/5ba0d7d2-9608-11e6-85b3-8113e8e2e06c.png)